internal/profile/profile_json.go

```diff
@@ -0,0 +1,56 @@
+// Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+// or more contributor license agreements. Licensed under the Elastic License;
+// you may not use this file except in compliance with the Elastic License.
+
+package profile
+
+import (
+	"encoding/json"
+	"os/user"
+	"path/filepath"
+	"time"
+
+	"github.com/pkg/errors"
+
+	"github.com/elastic/elastic-package/internal/version"
+)
+
+// Metadata stores the data associated with a given profile
+type Metadata struct {
+	Name        string
+	DateCreated time.Time `json:"date_created"`
+	User        string
+	Version     string `json:"elastic_package_version"`
+	From        string
+}
+
+// PackageProfileMetaFile is the filename of the profile metadata file
+const PackageProfileMetaFile ConfigFile = "profile.json"
+
+// CreateProfileMetadata creates the body of the profile.json file
+func CreateProfileMetadata(profileName string, profilePath string) (*SimpleFile, error) {
```

Comment:
WDYT about renaming this file to `metadata.go`?

Reply:
I'm trying to keep some kind of naming scheme with the files that define individual files, i.e, `static_kibana_config_yml.go` is `kibana.config.yml`.